lua/harpoon/mark.lua

```diff
@@ -115,6 +115,18 @@ local function validate_buf_name(buf_name)
     end
 end
 
+local function filter_filetype()
+    local current_filetype = vim.bo.filetype
+    local excluded_filetypes = harpoon.get_global_settings().excluded_filetypes
+    for filetype = 1, #excluded_filetypes do
+        if(current_filetype == excluded_filetypes[filetype] or current_filetype == "harpoon") then
```

Comment:
lets move the harpoon check outside of the if statement that way you don't have to do it on every loop.  I mean, it doesn't matter, but it matters to me :)

Reply:
That makes sense, I moved it out now with a different error message. Thanks Prime